index.js

```diff
@@ -830,6 +841,29 @@ const InstabugModule = {
     Instabug.reportScreenChange(screenName);
   },
 
+  /**
+   * Add experiments to next report.
+   * @param {string[]} experiments An array of experiments to add to the next report.
+   */
+  addExperiments(experiments) {
```

Comment:
This is the important part of the change, other changes in this file are because of the formatter (Prettier).

Reply:
I think we should exclude formatting-changes from this PR, and keep it until the code-style PR; unrelated changes makes it harder to use git blame and other version control inspection commands.

Reply:
Thanks, @TheBuggedYRN! I reverted all the formatting changes.